coral/src/app/features/approvals/schemas/SchemaApprovals.tsx

```diff
@@ -79,15 +90,44 @@ function SchemaApprovals() {
       />
     ) : undefined;
 
+  function approveRequest(req_no: number | null) {
+    console.log("approve", req_no);
+  }
+
+  function rejectRequest(req_no: number | null) {
+    console.log("approve", req_no);
+  }
+
   return (
-    <ApprovalsLayout
-      filters={filters}
-      table={table}
-      pagination={pagination}
-      isLoading={schemaRequestsIsLoading}
-      isErrorLoading={schemaRequestsIsError}
-      errorMessage={schemaRequestsError}
-    />
+    <>
+      {detailsModal.isOpen && (
+        <RequestDetailsModal
+          onClose={() => setDetailsModal({ isOpen: false, req_no: null })}
+          onApprove={() => {
+            approveRequest(detailsModal.req_no);
+          }}
+          onReject={() => {
+            setDetailsModal({ isOpen: false, req_no: null });
+            rejectRequest(detailsModal.req_no);
+          }}
+          isLoading={false}
```

Comment:
Something I completely forgot about my own PR with this Modal, is that we also need to think of the `disabledActions` case in other tables as well! As in, not showing the actions + disabling the Modal buttons if request is not `CREATED`. 

Just writing it down here so we don't forget to create an issue for it (or do it incidentally in the approve/reject flow issue)

Reply:
True! I'll make myself a note to think about that when adding the approve/reject for schemas ✍️